scalac-scoverage-runtime/shared/src/main/scala/scoverage/Invoker.scala

```diff
@@ -59,6 +66,78 @@ object Invoker {
     }
   }
 
+  /**
+   * We record that the given id has been invoked by appending its id to the coverage
+   * data file.
+   *
```

Comment:
This method (and method doc) duplicates a lot of the `def invoke` method. You should extract more of the method body in order to dedupe them.